src/main/java/com/internet/shop/controller/user/DeleteUserController.java

```diff
@@ -0,0 +1,22 @@
+package com.internet.shop.controller.user;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.interfaces.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
if we add creat shoppingCart for user in registration by default, it makes sense to delete whenever we delete user

Reply:
you're right